packages/s3-store/index.ts

```diff
@@ -42,6 +46,45 @@ function calcOffsetFromParts(parts?: Array<AWS.Part>) {
   return parts && parts.length > 0 ? parts.reduce((a, b) => a + b.Size, 0) : 0
 }
 
+function concatenateStreams(streams: Readable[]): Readable {
```

Comment:
Is it possible to use an existing module for that, e.g. https://www.npmjs.com/package/multistream? Then we would not have to maintain this logic.

Reply:
Ah nice! sure happy to use `multistream` in this case

Reply:
If this single function does the job, I personally don't see the reason to add more dependencies, unless there are other benefits.